lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -1826,6 +1864,93 @@ def cmd_db_rebuild_cache_help
     print_line
   end
 
+  def cmd_db_save_help
+    print_line "Usage: db_save <options> <name>"
+    print_line
+    print_line("   OPTIONS:")
+    print_line("       -d,--default           Set this data service as the default connection.")
```

Comment:
Minor: consider changing to "Set current data service as the default connection."